From: hyoyeong <164199371@qq.com>
Date: Fri, 28 Aug 2026 07:37:08 +0000
Subject: =?utf-8?b?VHJhbnNsYXRlZCB1c2luZyBXZWJsYXRlICjmnJ3pspzor60gKGtvX0tQ?=
 =?utf-8?b?KSk=?=

Currently translated at 100.0% (981 of 981 strings)

Translation: openkylin-huanghe/libpeony-qt
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/libpeony-qt/ko_KP/
---
 translations/libpeony-qt/libpeony-qt_ko_KP.ts      | 426 ++++++++++-----------
 .../peony-qt-desktop/peony-qt-desktop_ko_KP.ts     |  10 +-
 2 files changed, 218 insertions(+), 218 deletions(-)

diff --git a/translations/libpeony-qt/libpeony-qt_ko_KP.ts b/translations/libpeony-qt/libpeony-qt_ko_KP.ts
index 6414926..3aaa19c 100644
--- a/translations/libpeony-qt/libpeony-qt_ko_KP.ts
+++ b/translations/libpeony-qt/libpeony-qt_ko_KP.ts
@@ -954,17 +954,17 @@
     <message>
         <location filename="../../libpeony-qt/controls/property-page/permissions-properties-page.cpp" line="1058"/>
         <source>Add</source>
-        <translation>이미지 추가</translation>
+        <translation>추가</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/property-page/permissions-properties-page.cpp" line="1080"/>
         <source>Ok</source>
-        <translation>괜찮습니다</translation>
+        <translation>확인</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/property-page/permissions-properties-page.cpp" line="1081"/>
         <source>Cancel</source>
-        <translation>실행 취소</translation>
+        <translation>취소</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/property-page/permissions-properties-page.cpp" line="1286"/>
@@ -986,7 +986,7 @@
     <message>
         <location filename="../../libpeony-qt/controls/property-page/permissions-properties-page.cpp" line="1299"/>
         <source>User already exists in list</source>
-        <translation>사용자가 이미 목록에 존재합니다</translation>
+        <translation>사용자가 이미 리스트에 존재합니다</translation>
     </message>
 </context>
 <context>
@@ -999,7 +999,7 @@
     <message>
         <location filename="../../libpeony-qt/controls/property-page/open-with-properties-page.cpp" line="397"/>
         <source>Choose an Application to open this file</source>
-        <translation>이 파일을 열 응용 프로그람 선택</translation>
+        <translation>이 파일을 열 응용 프로그람을 선택하십시오</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/property-page/open-with-properties-page.cpp" line="404"/>
@@ -1009,12 +1009,12 @@
     <message>
         <location filename="../../libpeony-qt/controls/property-page/open-with-properties-page.cpp" line="412"/>
         <source>OK</source>
-        <translation>나중에 재시작</translation>
+        <translation>확인</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/property-page/open-with-properties-page.cpp" line="417"/>
         <source>Cancel</source>
-        <translation>실행 취소</translation>
+        <translation>취소</translation>
     </message>
 </context>
 <context>
@@ -1022,32 +1022,32 @@
     <message>
         <location filename="../../libpeony-qt/controls/property-page/basic-properties-page.cpp" line="386"/>
         <source>Choose a custom icon</source>
-        <translation>사용자 정의 아이콘 선택</translation>
+        <translation>사용자 지정 아이콘 선택</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/property-page/basic-properties-page.cpp" line="307"/>
         <source>Type:</source>
-        <translation>장치 유형:</translation>
+        <translation>유형:</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/property-page/basic-properties-page.cpp" line="251"/>
         <source>Name</source>
-        <translation>블루투스 장치 이름</translation>
+        <translation>이름</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/property-page/basic-properties-page.cpp" line="252"/>
         <source>Location</source>
-        <translation>위치 정보 허용</translation>
+        <translation>위치</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/property-page/basic-properties-page.cpp" line="559"/>
         <source>symbolLink</source>
-        <translation>바로가기</translation>
+        <translation>바로 가기</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/property-page/basic-properties-page.cpp" line="563"/>
         <source>Folder</source>
-        <translation>문서함</translation>
+        <translation>폴더</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/property-page/basic-properties-page.cpp" line="314"/>
@@ -1194,7 +1194,7 @@
     <message>
         <location filename="../../libpeony-qt/convenient-utils/clipboard-utils.cpp" line="357"/>
         <source>Can not cut: The source file and the target file are the same</source>
-        <translation>잘라내기 할 수 없습니다: 원본 파일과 대상 파일이 동일합니다</translation>
+        <translation>잘라낼 수 없습니다: 원본 파일과 대상 파일이 동일합니다</translation>
     </message>
 </context>
 <context>
@@ -1300,7 +1300,7 @@
         <location filename="../../libpeony-qt/controls/property-page/computer-properties-page.cpp" line="309"/>
         <location filename="../../libpeony-qt/controls/property-page/computer-properties-page.cpp" line="450"/>
         <source>Type: </source>
-        <translation>유형: </translation>
+        <translation>파일 체계: </translation>
     </message>
     <message>
         <source>Share Data</source>
@@ -1323,7 +1323,7 @@
     <message>
         <location filename="../../libpeony-qt/controls/property-page/computer-properties-page.cpp" line="488"/>
         <source>Unknown</source>
-        <translation>알 수 없는</translation>
+        <translation>알 수 없는 파티션</translation>
     </message>
 </context>
 <context>
@@ -1341,12 +1341,12 @@
     <message>
         <location filename="../../libpeony-qt/connect-to-server-dialog.cpp" line="188"/>
         <source>Port</source>
-        <translation>구단</translation>
+        <translation>포트</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/connect-to-server-dialog.cpp" line="189"/>
         <source>Type</source>
-        <translation>네트워크 카드 유형</translation>
+        <translation>유형</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/connect-to-server-dialog.cpp" line="203"/>
@@ -1356,7 +1356,7 @@
     <message>
         <location filename="../../libpeony-qt/connect-to-server-dialog.cpp" line="206"/>
         <source>Default</source>
-        <translation>기본으로 설정</translation>
+        <translation>기본값</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/connect-to-server-dialog.cpp" line="230"/>
@@ -1366,17 +1366,17 @@
     <message>
         <location filename="../../libpeony-qt/connect-to-server-dialog.cpp" line="243"/>
         <source>Add</source>
-        <translation>이미지 추가</translation>
+        <translation>추가</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/connect-to-server-dialog.cpp" line="244"/>
         <source>Delete</source>
-        <translation>삭제 표시</translation>
+        <translation>삭제</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/connect-to-server-dialog.cpp" line="245"/>
         <source>Connect</source>
-        <translation>스캐너 련결</translation>
+        <translation>련결</translation>
     </message>
     <message>
         <source>Ip input error, please re-enter!</source>
@@ -1411,17 +1411,17 @@
     <message>
         <location filename="../../libpeony-qt/connect-to-server-dialog.cpp" line="628"/>
         <source>Guest</source>
-        <translation>게스트 로그인</translation>
+        <translation>게스트 (익명 로그인)</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/connect-to-server-dialog.cpp" line="646"/>
         <source>Name</source>
-        <translation>블루투스 장치 이름</translation>
+        <translation>이름</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/connect-to-server-dialog.cpp" line="647"/>
         <source>Password</source>
-        <translation>키</translation>
+        <translation>비밀번호</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/connect-to-server-dialog.cpp" line="649"/>
@@ -1431,12 +1431,12 @@
     <message>
         <location filename="../../libpeony-qt/connect-to-server-dialog.cpp" line="674"/>
         <source>Cancel</source>
-        <translation>실행 취소</translation>
+        <translation>취소</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/connect-to-server-dialog.cpp" line="675"/>
         <source>OK</source>
-        <translation>나중에 재시작</translation>
+        <translation>확인</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/connect-to-server-dialog.cpp" line="629"/>
@@ -1481,7 +1481,7 @@
     <message>
         <location filename="../../libpeony-qt/controls/menu/menu-plugin-manager.cpp" line="337"/>
         <source>Create Link to Desktop</source>
-        <translation>바탕 화면 바로가기로 보내기</translation>
+        <translation>바탕 화면 바로 가기로 보내기</translation>
     </message>
     <message>
         <source>Peony-Qt Share File menu Extension</source>
@@ -1549,17 +1549,17 @@
     <message>
         <location filename="../../libpeony-qt/controls/property-page/basic-properties-page.cpp" line="1977"/>
         <source>Select an icon from the system icons:</source>
-        <translation>시스템 아이콘 목록에서 아이콘 을 선택하십시오:</translation>
+        <translation>체계 아이콘 목록에서 아이콘을 선택하십시오:</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/property-page/basic-properties-page.cpp" line="1978"/>
         <source>Custom Icons:</source>
-        <translation>사용자 정의 아이콘:</translation>
+        <translation>사용자 지정 아이콘:</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/property-page/basic-properties-page.cpp" line="1996"/>
         <source>Click Browse to load a custom icon</source>
-        <translation>찾아보기를 클릭하여 사용자 정의 아이콘을 불러오십시오</translation>
+        <translation>찾아보기를 클릭하여 사용자 지정 아이콘을 불러오십시오</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/property-page/basic-properties-page.cpp" line="1998"/>
@@ -1574,12 +1574,12 @@
     <message>
         <location filename="../../libpeony-qt/controls/property-page/basic-properties-page.cpp" line="2008"/>
         <source>Cancel</source>
-        <translation>실행 취소</translation>
+        <translation>취소</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/property-page/basic-properties-page.cpp" line="2009"/>
         <source>Ok</source>
-        <translation>괜찮습니다</translation>
+        <translation>확인</translation>
     </message>
 </context>
 <context>
@@ -1630,7 +1630,7 @@
     <message>
         <location filename="../../libpeony-qt/controls/property-page/details-properties-page.cpp" line="181"/>
         <source>Name:</source>
-        <translation>애플리케이션 이름:</translation>
+        <translation>이름:</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/property-page/details-properties-page.cpp" line="184"/>
@@ -1690,7 +1690,7 @@
     <message>
         <location filename="../../libpeony-qt/controls/property-page/details-properties-page.cpp" line="261"/>
         <source>Computer</source>
-        <translation>나의 컴퓨터</translation>
+        <translation>컴퓨터</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/property-page/details-properties-page.cpp" line="262"/>
@@ -1705,7 +1705,7 @@
     <message>
         <location filename="../../libpeony-qt/controls/property-page/details-properties-page.cpp" line="325"/>
         <source>Unknown</source>
-        <translation>알 수 없는</translation>
+        <translation>알 수 없는 파티션</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/property-page/details-properties-page.cpp" line="373"/>
@@ -1731,7 +1731,7 @@
     <message>
         <location filename="../../libpeony-qt/controls/directory-view/view/icon-view/icon-view.cpp" line="392"/>
         <source>This operation is not supported.</source>
-        <translation>이 작업은 지원되지 않습니다.</translation>
+        <translation>이 조작은 지원되지 않습니다.</translation>
     </message>
     <message>
         <source>Icon View</source>
@@ -1748,7 +1748,7 @@
     <message>
         <location filename="../../libpeony-qt/controls/directory-view/view/list-view/list-view.cpp" line="826"/>
         <source>This operation is not supported.</source>
-        <translation>이 작업은 지원되지 않습니다.</translation>
+        <translation>이 조작은 지원되지 않습니다.</translation>
     </message>
     <message>
         <source>List View</source>
@@ -1760,22 +1760,22 @@
     <message>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="520"/>
         <source>Add to bookmark</source>
-        <translation>빠른 액세스에 추가</translation>
+        <translation>빠른 접근에 추가</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="751"/>
         <source>New Folder</source>
-        <translation>폴더</translation>
+        <translation>새 폴더</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="961"/>
         <source>Name</source>
-        <translation>블루투스 장치 이름</translation>
+        <translation>파일 이름</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="963"/>
         <source>File Type</source>
-        <translation>파일 형식</translation>
+        <translation>파일 유형</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="964"/>
@@ -1799,7 +1799,7 @@
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="547"/>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="646"/>
         <source>Open</source>
-        <translation>선택</translation>
+        <translation>열기</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="432"/>
@@ -1809,13 +1809,13 @@
     <message>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="432"/>
         <source>Open directory failed, you have no permission!</source>
-        <translation>폴더 열기 실패, 해당 디렉토리의 권한이 없습니다!</translation>
+        <translation>폴더 열기 실패, 해당 권한이 없습니다!</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="465"/>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="571"/>
         <source>Open with...</source>
-        <translation>다른 방법으로 열기...</translation>
+        <translation>열기 방식...</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="480"/>
@@ -1850,7 +1850,7 @@
     <message>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="747"/>
         <source>Folder</source>
-        <translation>문서함</translation>
+        <translation>폴더</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="759"/>
@@ -1865,7 +1865,7 @@
     <message>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="954"/>
         <source>Sort By</source>
-        <translation>배열 방식</translation>
+        <translation>정렬 방식</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="965"/>
@@ -1885,7 +1885,7 @@
     <message>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="1769"/>
         <source>Mount</source>
-        <translation>탑재</translation>
+        <translation>마운트</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="2008"/>
@@ -1918,12 +1918,12 @@
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="1319"/>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="1617"/>
         <source>Copy</source>
-        <translation>본지에 복사</translation>
+        <translation>복사</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="1644"/>
         <source>Cancel</source>
-        <translation>실행 취소</translation>
+        <translation>취소</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="1751"/>
@@ -1949,7 +1949,7 @@
     <message>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="1063"/>
         <source>Refresh</source>
-        <translation>새 고침</translation>
+        <translation>새로 고침</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="1346"/>
@@ -1972,7 +1972,7 @@
     <message>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="1661"/>
         <source>Restore</source>
-        <translation>열기</translation>
+        <translation>복원</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="1224"/>
@@ -1980,7 +1980,7 @@
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="1624"/>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="1642"/>
         <source>Delete</source>
-        <translation>삭제 표시</translation>
+        <translation>삭제</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="962"/>
@@ -1998,27 +1998,27 @@
     <message>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="1008"/>
         <source>Newest to oldest</source>
-        <translation>새것부터 오래된 것까지</translation>
+        <translation>최신순</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="1009"/>
         <source>Oldest to newest</source>
-        <translation>오래된 것부터 새것까지</translation>
+        <translation>오래된 순</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="1012"/>
         <source>Files from large to small</source>
-        <translation>큰것부터 작은 것까지</translation>
+        <translation>크기 내림차순</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="1013"/>
         <source>Files from small to large</source>
-        <translation>작은 것부터 큰것까지</translation>
+        <translation>크기 오름차순</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="1029"/>
         <source>Sort Preferences</source>
-        <translation>정렬 기본 설정</translation>
+        <translation>정렬 선호</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="1750"/>
@@ -2036,7 +2036,7 @@
     <message>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="1171"/>
         <source>Rename</source>
-        <translation>블루투스 이름 변경</translation>
+        <translation>이름 바꾸기</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="1358"/>
@@ -2056,7 +2056,7 @@
     <message>
         <location filename="../../libpeony-qt/controls/menu/directory-view-menu/directory-view-menu.cpp" line="1721"/>
         <source>Open Parent Folder in New Window</source>
-        <translation>새 창에서 파일이 들어있는 디렉토리 열기</translation>
+        <translation>새 창에서 파일이 위치한 폴더 열기</translation>
     </message>
 </context>
 <context>
@@ -2074,12 +2074,12 @@
     <message>
         <location filename="../../libpeony-qt/extensions-manager-widget.cpp" line="78"/>
         <source>Ok</source>
-        <translation>괜찮습니다</translation>
+        <translation>확인</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/extensions-manager-widget.cpp" line="80"/>
         <source>Cancel</source>
-        <translation>실행 취소</translation>
+        <translation>취소</translation>
     </message>
 </context>
 <context>
@@ -2117,7 +2117,7 @@
     <message>
         <location filename="../../libpeony-qt/windows/fm-window.cpp" line="435"/>
         <source>New Folder</source>
-        <translation>폴더</translation>
+        <translation>새 폴더</translation>
     </message>
 </context>
 <context>
@@ -2130,7 +2130,7 @@
     <message>
         <location filename="../../libpeony-qt/file-operation/file-batch-rename-operation.cpp" line="76"/>
         <source>Invalid file name %1%2%3 .</source>
-        <translation>부적절한 파일명 %1%2%3.</translation>
+        <translation>유효하지 않은 파일명 %1%2%3.</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-batch-rename-operation.cpp" line="91"/>
@@ -2160,7 +2160,7 @@
     <message>
         <location filename="../../libpeony-qt/file-copy.cpp" line="248"/>
         <source>Error when copy file: %1, can not copy special files, skip this file and continue?</source>
-        <translation>파일복사:%1시 오류 발생, 특수 형식 파일은 복사 할 수 없습니다. 이 파일을 건너뛰고 계속 진행하시겠습니까?</translation>
+        <translation>파일 복사:%1시 오류 발생, 특수 형식 파일은 복사할 수 없습니다. 이 파일을 건너뛰고 계속 진행하시겠습니까?</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-copy.cpp" line="285"/>
@@ -2175,7 +2175,7 @@
     <message>
         <location filename="../../libpeony-qt/file-copy.cpp" line="332"/>
         <source>Vfat/FAT32 file systems do not support a single file that occupies more than 4 GB space!</source>
-        <translation>vfat/fat32 파일 체계은 단일 파일 크기가 4GB를 초과하는 것을 지원하지 않습니다!</translation>
+        <translation>vfat/fat32 파일 체계는 단일 파일 크기가 4GB를 초과하는 것을 지원하지 않습니다!</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-copy.cpp" line="388"/>
@@ -2185,7 +2185,7 @@
     <message>
         <location filename="../../libpeony-qt/file-copy.cpp" line="566"/>
         <source>Failed to create %1. Please ensure if it is in root directory, or if the device supports gphoto2 protocol correctly.</source>
-        <translation>파일 %1 생성 실패, 루트 디렉토리에서 작업 중인지 혹은 장치가 gphoto2 프로토콜을 올바르게 지원하는지 확인하십시오.</translation>
+        <translation>파일 %1 생성 실패, 루트 디렉토리에서 작업 중인지 또는 장치가 gphoto2 프로토콜을 올바르게 지원하는지 확인하십시오.</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-copy.cpp" line="572"/>
@@ -2211,7 +2211,7 @@
         <location filename="../../libpeony-qt/file-copy.cpp" line="379"/>
         <location filename="../../libpeony-qt/file-copy.cpp" line="479"/>
         <source>operation cancel</source>
-        <translation>작업 취소</translation>
+        <translation>조작 취소</translation>
     </message>
 </context>
 <context>
@@ -2219,7 +2219,7 @@
     <message>
         <location filename="../../libpeony-qt/file-operation/file-copy-operation.cpp" line="247"/>
         <source>Create folder %1 failed: %2</source>
-        <translation>디렉토리 %1 생성 실패: %2</translation>
+        <translation>폴더 %1 생성 실패: %2</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-copy-operation.cpp" line="251"/>
@@ -2260,7 +2260,7 @@
     <message>
         <location filename="../../libpeony-qt/file-operation/file-copy-operation.cpp" line="996"/>
         <source>Can not copy %1 to %2: Read-only mode, can not write-in</source>
-        <translation>%1을(를) %2에 복사할 수 없습니다：읽기 전용 모드라 쓸 수 없습니다</translation>
+        <translation>%1을(를) %2에 복사할 수 없습니다：읽기 전용 모드이므로 쓸 수 없습니다</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-copy-operation.cpp" line="1011"/>
@@ -2312,7 +2312,7 @@
     <message>
         <location filename="../../libpeony-qt/file-enumerator.cpp" line="636"/>
         <source>The password dialog box is canceled</source>
-        <translation>비밀번호 대화 상자가 취소 되었습니다</translation>
+        <translation>비밀번호 대화 상자가 취소되었습니다</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-enumerator.cpp" line="638"/>
@@ -2371,7 +2371,7 @@
     <message>
         <location filename="../../libpeony-qt/model/file-item.cpp" line="281"/>
         <source>Warning</source>
-        <translation>알림</translation>
+        <translation>경고</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/model/file-item.cpp" line="283"/>
@@ -2384,17 +2384,17 @@
     <message>
         <location filename="../../libpeony-qt/model/file-item.cpp" line="363"/>
         <source>Open Link failed</source>
-        <translation>바로 가기 열기 실패</translation>
+        <translation>바로 가기 링크 실패</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/model/file-item.cpp" line="364"/>
         <source>File not exist, do you want to delete the link file?</source>
-        <translation>대상 폴더가 존재하지 않습니다, 해당 무효 바로 가기을 삭제하시겠습니까?</translation>
+        <translation>대상 폴더가 존재하지 않습니다. 해당 무효 바로 가기를 삭제하시겠습니까?</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/model/file-item.cpp" line="378"/>
         <source>Can not open path &quot;%1&quot;，permission denied.</source>
-        <translation>경로 &quot;%1&quot; 열기 실패, 권한이 거부되었습니다.</translation>
+        <translation>경로 “%1” 열기 실패, 권한이 거부되었습니다.</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/model/file-item.cpp" line="388"/>
@@ -2412,7 +2412,7 @@
     <message>
         <location filename="../../libpeony-qt/model/file-item-model.cpp" line="383"/>
         <source>Symbol Link, </source>
-        <translation>바로가기, </translation>
+        <translation>바로 가기, </translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/model/file-item-model.cpp" line="309"/>
@@ -2452,7 +2452,7 @@
     <message>
         <location filename="../../libpeony-qt/model/file-item-model.cpp" line="472"/>
         <source>Path</source>
-        <translation>경로 선택</translation>
+        <translation>경로</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/model/file-item-model.cpp" line="815"/>
@@ -2462,12 +2462,12 @@
     <message>
         <location filename="../../libpeony-qt/model/file-item-model.cpp" line="821"/>
         <source>Operation not supported</source>
-        <translation>작업이 지원되지 않습니다</translation>
+        <translation>조작이 지원되지 않습니다</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/model/file-item-model.cpp" line="465"/>
         <source>File Type</source>
-        <translation>파일 형식</translation>
+        <translation>파일 유형</translation>
     </message>
 </context>
 <context>
@@ -2485,7 +2485,7 @@
     <message>
         <location filename="../../libpeony-qt/controls/menu/menu-plugin-manager.cpp" line="284"/>
         <source>Label Management ...</source>
-        <translation>표식 관리 ...</translation>
+        <translation>표시 관리 ...</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/menu-plugin-manager.cpp" line="297"/>
@@ -2511,7 +2511,7 @@
     <message>
         <location filename="../../libpeony-qt/controls/menu/menu-plugin-manager.cpp" line="490"/>
         <source>Label Management ...</source>
-        <translation>표식 관리 ...</translation>
+        <translation>표시 관리 ...</translation>
     </message>
 </context>
 <context>
@@ -2545,7 +2545,7 @@
     <message>
         <location filename="../../libpeony-qt/file-launcher/file-lauch-dialog.cpp" line="182"/>
         <source>known</source>
-        <translation>알려진</translation>
+        <translation>기지</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-launcher/file-lauch-dialog.cpp" line="185"/>
@@ -2591,13 +2591,13 @@
     <message>
         <location filename="../../libpeony-qt/file-launcher/file-lauch-dialog.cpp" line="377"/>
         <source>Ok</source>
-        <translation>괜찮습니다</translation>
+        <translation>확인</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-launcher/file-lauch-dialog.cpp" line="378"/>
         <location filename="../../libpeony-qt/file-launcher/file-lauch-dialog.cpp" line="456"/>
         <source>Cancel</source>
-        <translation>실행 취소</translation>
+        <translation>취소</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-launcher/file-lauch-dialog.cpp" line="447"/>
@@ -2676,7 +2676,7 @@
         <location filename="../../libpeony-qt/file-launcher/file-launch-action.cpp" line="385"/>
         <location filename="../../libpeony-qt/file-launcher/file-launch-action.cpp" line="588"/>
         <source>Open Link failed</source>
-        <translation>바로 가기 열기 실패</translation>
+        <translation>링크 열기 실패</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-launcher/file-launch-action.cpp" line="386"/>
@@ -2687,12 +2687,12 @@
     <message>
         <location filename="../../libpeony-qt/file-launcher/file-launch-action.cpp" line="389"/>
         <source>Delete</source>
-        <translation>삭제 표시</translation>
+        <translation>삭제</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-launcher/file-launch-action.cpp" line="390"/>
         <source>Cancel</source>
-        <translation>실행 취소</translation>
+        <translation>취소</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-launcher/file-launch-action.cpp" line="404"/>
@@ -2711,8 +2711,8 @@
         <location filename="../../libpeony-qt/file-launcher/file-launch-action.cpp" line="606"/>
         <source>The linked app is changed or uninstalled, so it can not work correctly. 
 Do you want to delete the link file?</source>
-        <translation>해당 링크가 가리키는 응용 프로그람이 수정되었거나 제거되어 이 바로가기가 정상적으로 작동하지 않습니다.
-이 바로가기를 삭제하시겠습니까?</translation>
+        <translation>해당 링크가 가리키는 응용 프로그람이 수정되었거나 제거되어 이 바로 가기가 정상적으로 작동하지 않습니다.
+이 바로 가기를 삭제하시겠습니까?</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-launcher/file-launch-action.cpp" line="427"/>
@@ -2733,7 +2733,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/file-launcher/file-launch-action.cpp" line="700"/>
         <source>Operate Tips</source>
-        <translation>작업 안내</translation>
+        <translation>안내</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-launcher/file-launch-action.cpp" line="716"/>
@@ -2750,12 +2750,12 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/file-launcher/file-launch-action.cpp" line="746"/>
         <source>Sorry, the system currently does not have a default application set for opening %1 file format.</source>
-        <translation>죄송합니다. 현재 시스템에 %1 파일 형식을 열 응용 프로그람이 설정되지 않았습니다.</translation>
+        <translation>죄송합니다. 현재 체계에 %1 파일 형식을 열 응용 프로그람이 설정되지 않았습니다.</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-launcher/file-launch-action.cpp" line="747"/>
         <source>Please search the software store for applications that can open this file, or select an existing application in the system.</source>
-        <translation>소프트웨어 상점에서 이 파일을 열수 있는 응용 프로그람을 검색하거나 체계에 있는 응용 프로그람을 선택하십시오.</translation>
+        <translation>소프트웨어 상점에서 이 파일을 열 수 있는 응용 프로그람을 검색하거나 체계에 있는 응용 프로그람을 선택하십시오.</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-launcher/file-launch-action.cpp" line="754"/>
@@ -2841,12 +2841,12 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-move-operation.cpp" line="1970"/>
         <source>Can not move %1 to %2: Read-only mode, can not write-in</source>
-        <translation>%1을(를) %2로 이동할 수 없습니다: 읽기 전용 모드로 쓸 수 없습니다</translation>
+        <translation>%1을(를) %2로 이동할 수 없습니다: 읽기 전용 모드이므로 쓸 수 없습니다</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-move-operation.cpp" line="1985"/>
         <source>Can not move %1 to %2: Permission denied</source>
-        <translation>%1을(를) %2로 이동할수없습니다: 권한 부족</translation>
+        <translation>%1을(를) %2로 이동할 수 없습니다: 권한 부족</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-move-operation.cpp" line="946"/>
@@ -2884,7 +2884,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-move-operation.cpp" line="2223"/>
         <source>Burning does not support replacement</source>
-        <translation>굽기 작업은 대체를 지원하지 않습니다</translation>
+        <translation>굽기 조작은 대체를 지원하지 않습니다</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-move-operation.cpp" line="906"/>
@@ -2937,7 +2937,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-error-dialog.cpp" line="63"/>
         <source>Ignore</source>
-        <translation>메시지 무시</translation>
+        <translation>무시</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-error-dialog.cpp" line="64"/>
@@ -2980,7 +2980,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-error-dialog-base.cpp" line="89"/>
         <source>Close</source>
-        <translation>끄기</translation>
+        <translation>닫기</translation>
     </message>
 </context>
 <context>
@@ -2994,7 +2994,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-error-dialogs.cpp" line="120"/>
         <source>Ignore</source>
-        <translation>메시지 무시</translation>
+        <translation>무시</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-error-dialogs.cpp" line="139"/>
@@ -3014,7 +3014,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-error-dialogs.cpp" line="173"/>
         <source>Unexpected error from %1 to %2</source>
-        <translation>%1부터 %2까지 조작 중 예외 오류 발생</translation>
+        <translation>%1부터 %2까지 조작 시 예치지 못한 오류 발생</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-error-dialogs.cpp" line="192"/>
@@ -3032,17 +3032,17 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-error-dialogs.cpp" line="581"/>
         <source>Cancel</source>
-        <translation>실행 취소</translation>
+        <translation>취소</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-error-dialogs.cpp" line="571"/>
         <source>Delete</source>
-        <translation>삭제 표시</translation>
+        <translation>삭제</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-error-dialogs.cpp" line="617"/>
         <source>Make sure the disk is not full or write protected and that the file is not protected</source>
-        <translation>디스크가 가득 차지 않았거나 쓰기 금지 상태가 아니며 파일이 사용 중이지 않은지 확인하십시오</translation>
+        <translation>디스크 공간 부족, 쓰기 보호 또는 파일 사용 여부를 확인하십시오</translation>
     </message>
 </context>
 <context>
@@ -3050,17 +3050,17 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-error-dialogs.cpp" line="495"/>
         <source>OK</source>
-        <translation>나중에 재시작</translation>
+        <translation>확인</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-error-dialogs.cpp" line="504"/>
         <source>Cancel</source>
-        <translation>실행 취소</translation>
+        <translation>취소</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-error-dialogs.cpp" line="533"/>
         <source>Make sure the disk is not full or write protected and that the file is not protected</source>
-        <translation>디스크가 가득 차지 않았거나 쓰기 금지 상태가 아니며 파일이 사용 중이지 않은지 확인하십시오</translation>
+        <translation>디스크 공간 부족, 쓰기 보호 또는 파일 사용 중 여부를 확인하십시오</translation>
     </message>
 </context>
 <context>
@@ -3073,7 +3073,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-error-dialogs.cpp" line="395"/>
         <source>Cancel</source>
-        <translation>실행 취소</translation>
+        <translation>취소</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-error-dialogs.cpp" line="402"/>
@@ -3083,7 +3083,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-error-dialogs.cpp" line="437"/>
         <source>Make sure the disk is not full or write protected and that the file is not protected</source>
-        <translation>디스크가 가득 차지 않았거나 쓰기 금지 상태가 아니며 파일이 사용 중이지 않은지 확인하십시오</translation>
+        <translation>디스크 공간 부족, 쓰기 보호 또는 파일 사용 중 여부를 확인하십시오</translation>
     </message>
 </context>
 <context>
@@ -3091,12 +3091,12 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-error-dialogs.cpp" line="302"/>
         <source>OK</source>
-        <translation>나중에 재시작</translation>
+        <translation>확인</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-error-dialogs.cpp" line="334"/>
         <source>Make sure the disk is not full or write protected and that the file is not protected</source>
-        <translation>디스크가 가득 차지 않았거나 쓰기 금지 상태가 아니며 파일이 사용 중이지 않은지 확인하십시오</translation>
+        <translation>디스크 공간 부족, 쓰기 보호 또는 파일 사용 여부를 확인하십시오</translation>
     </message>
 </context>
 <context>
@@ -3110,7 +3110,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-manager.cpp" line="1385"/>
         <source>Copy</source>
-        <translation>본지에 복사</translation>
+        <translation>복사</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-manager.cpp" line="1388"/>
@@ -3120,17 +3120,17 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-manager.cpp" line="1393"/>
         <source>Rename</source>
-        <translation>블루투스 이름 변경</translation>
+        <translation>이름 바꾸기</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-manager.cpp" line="1396"/>
         <source>Link</source>
-        <translation>련결</translation>
+        <translation>링크</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-manager.cpp" line="1399"/>
         <source>Delete</source>
-        <translation>삭제 표시</translation>
+        <translation>삭제</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-manager.cpp" line="1402"/>
@@ -3140,7 +3140,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-manager.cpp" line="1405"/>
         <source>Restore</source>
-        <translation>열기</translation>
+        <translation>복원</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-manager.cpp" line="1410"/>
@@ -3158,7 +3158,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-manager.cpp" line="332"/>
         <source>&apos;%1&apos; is occupied，you cannot operate!</source>
-        <translation>&quot;%1&quot;이(가) 이미 사용 중이므로 작업을 진행할 수 없습니다!</translation>
+        <translation>&quot;%1&quot;이(가) 이미 사용 중이므로 조작을 진행할 수 없습니다!</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-manager.cpp" line="347"/>
@@ -3178,19 +3178,19 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-manager.cpp" line="755"/>
         <source>OK</source>
-        <translation>나중에 재시작</translation>
+        <translation>확인</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-manager.cpp" line="353"/>
         <location filename="../../libpeony-qt/file-operation/file-operation-manager.cpp" line="379"/>
         <source>Cancel</source>
-        <translation>실행 취소</translation>
+        <translation>취소</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-manager.cpp" line="351"/>
         <location filename="../../libpeony-qt/file-operation/file-operation-manager.cpp" line="377"/>
         <source>Delete</source>
-        <translation>삭제 표시</translation>
+        <translation>삭제</translation>
     </message>
     <message>
         <source>Do you want to put selected %1 item(s) into trash?</source>
@@ -3199,7 +3199,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-manager.cpp" line="382"/>
         <source>Do not show again</source>
-        <translation>다시 보지 않기</translation>
+        <translation>다시 표시하지 않기</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-manager.cpp" line="537"/>
@@ -3224,12 +3224,12 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-manager.cpp" line="594"/>
         <source>Can&apos;t delete.</source>
-        <translation>삭제 할 수 없음.</translation>
+        <translation>삭제할 수 없습니다.</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-manager.cpp" line="595"/>
         <source>You can&apos;t delete a file whenthe file is doing another operation</source>
-        <translation>다른 작업을 진행중인 파일은 삭제할 수 없습니다</translation>
+        <translation>다른 작업을 진행 중인 파일은 삭제할 수 없습니다</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-manager.cpp" line="725"/>
@@ -3244,7 +3244,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-manager.cpp" line="753"/>
         <source>The long name file is saved to %1</source>
-        <translation>긴 파일이 %1에 보관되었습니다</translation>
+        <translation>긴 파일이 %1에 저장되었습니다</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-manager.cpp" line="776"/>
@@ -3254,7 +3254,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-manager.cpp" line="777"/>
         <source>ok</source>
-        <translation>확정</translation>
+        <translation>확인</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-manager.cpp" line="778"/>
@@ -3264,7 +3264,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-manager.cpp" line="1189"/>
         <source>Undo %1</source>
-        <translation>%1 취소</translation>
+        <translation>%1 실행 취소</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-manager.cpp" line="1197"/>
@@ -3351,7 +3351,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-progress-wizard.cpp" line="160"/>
         <source>%1 files, %2</source>
-        <translation>%1개 파일, 총%2</translation>
+        <translation>%1개 파일, 총 %2</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-operation-progress-wizard.cpp" line="257"/>
@@ -3379,17 +3379,17 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/controls/preview-page/default-preview-page/default-preview-page.cpp" line="326"/>
         <source>Location</source>
-        <translation>위치 정보 허용</translation>
+        <translation>위치</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/preview-page/default-preview-page/default-preview-page.cpp" line="334"/>
         <source>File Type</source>
-        <translation>파일 형식</translation>
+        <translation>파일 유형</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/preview-page/default-preview-page/default-preview-page.cpp" line="342"/>
         <source>Size</source>
-        <translation>글꼴 크기</translation>
+        <translation>크기</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/preview-page/default-preview-page/default-preview-page.cpp" line="350"/>
@@ -3404,7 +3404,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/controls/preview-page/default-preview-page/default-preview-page.cpp" line="366"/>
         <source>Time Access</source>
-        <translation>방문 시간</translation>
+        <translation>접근 시간</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/preview-page/default-preview-page/default-preview-page.cpp" line="376"/>
@@ -3439,17 +3439,17 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/controls/preview-page/default-preview-page/default-preview-page.cpp" line="458"/>
         <source>Cancel</source>
-        <translation>실행 취소</translation>
+        <translation>취소</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/preview-page/default-preview-page/default-preview-page.cpp" line="459"/>
         <source>OK</source>
-        <translation>나중에 재시작</translation>
+        <translation>확인</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/preview-page/default-preview-page/default-preview-page.cpp" line="465"/>
         <source>&lt;font color=&apos;gray&apos;&gt;After removal,when files in the space are aggregated and updated,these files will be no longer be displayed&lt;/font&gt;</source>
-        <translation>&lt;font color=&apos;gray&apos;&gt;제거한 후 선별 조건에 부합되는 파일이 실시간으로 업데이트될 때, 제거된 파일이 더 이상 해당 공간에표시되지 않습니다&lt;/font&gt;</translation>
+        <translation>&lt;font color=&apos;gray&apos;&gt;제거한 후 선별 조건에 부합되는 파일이 실시간으로 업데이트될 때, 제거된 파일이 더 이상 해당 공간에 표시되지 않습니다&lt;/font&gt;</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/preview-page/default-preview-page/default-preview-page.cpp" line="466"/>
@@ -3464,7 +3464,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/controls/preview-page/default-preview-page/default-preview-page.cpp" line="470"/>
         <source>Do not ask again</source>
-        <translation>다시 묻지 않기</translation>
+        <translation>다시 알리지 않기</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/preview-page/default-preview-page/default-preview-page.cpp" line="554"/>
@@ -3481,7 +3481,7 @@ Do you want to delete the link file?</source>
         <location filename="../../libpeony-qt/controls/preview-page/default-preview-page/default-preview-page.cpp" line="718"/>
         <location filename="../../libpeony-qt/controls/preview-page/default-preview-page/default-preview-page.cpp" line="719"/>
         <source>%1 total, %2 hidden</source>
-        <translation>총 %1개 항목, 그 중 %2개는 숨김 파일</translation>
+        <translation>총 %1개 항목, 그 중 %2개는 숨김 파일입니다</translation>
     </message>
 </context>
 <context>
@@ -3494,7 +3494,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-rename-operation.cpp" line="105"/>
         <source>Invalid file name %1%2%3 .</source>
-        <translation>부적절한 파일명 %1%2%3.</translation>
+        <translation>유효하지 않은 파일명 %1%2%3.</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-rename-operation.cpp" line="121"/>
@@ -3504,7 +3504,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-rename-operation.cpp" line="155"/>
         <source>When change the file suffix, the file may be invalid. Are you sure to change it ?</source>
-        <translation>파일 확장명를 변경하면 파일을 사용하지 못 할수도 있습니다. 정말 변경하시겠습니까?</translation>
+        <translation>파일 확장명를 변경하면 파일을 사용하지 못할 수도 있습니다. 정말 변경하시겠습니까?</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/file-operation/file-rename-operation.cpp" line="331"/>
@@ -3612,7 +3612,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/windows/properties-window.cpp" line="1118"/>
         <source>Close</source>
-        <translation>끄기</translation>
+        <translation>닫기</translation>
     </message>
 </context>
 <context>
@@ -3633,14 +3633,14 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="495"/>
         <source>OK</source>
-        <translation>나중에 재시작</translation>
+        <translation>확인</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="177"/>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="363"/>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="494"/>
         <source>Cancel</source>
-        <translation>실행 취소</translation>
+        <translation>취소</translation>
     </message>
     <message>
         <source>Rename Space</source>
@@ -3663,12 +3663,12 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="139"/>
         <source>Close dynamically</source>
-        <translation>자동 업데이트 취소</translation>
+        <translation>자동 업데이트 비활성화</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="141"/>
         <source>Open dynamically</source>
-        <translation>자동 업데이트 열기</translation>
+        <translation>자동 업데이트 활성화</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="148"/>
@@ -3682,17 +3682,17 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="189"/>
         <source>Open</source>
-        <translation>선택</translation>
+        <translation>열기</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="202"/>
         <source>Open With</source>
-        <translation>련결 방식</translation>
+        <translation>열기 방식</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="223"/>
         <source>More applications</source>
-        <translation>더 많은 응용 프로그람</translation>
+        <translation>더 많은 응용</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="229"/>
@@ -3702,18 +3702,18 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="278"/>
         <source>Add File</source>
-        <translation>추가</translation>
+        <translation>파일 추가</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="283"/>
         <source>Choose file to add into space</source>
-        <translation>공간에 파일 추가</translation>
+        <translation>공간에 추가할 파일 선택</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="299"/>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="341"/>
         <source>File %1 is existed in space, please not add file repeatly.</source>
-        <translation>공간에 이미 파일 %1이(가) 존재합니다.반복하여 추가할 수 없습니다.</translation>
+        <translation>공간에 이미 파일 %1이(가) 존재합니다. 중복 추가할 수 없습니다.</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="314"/>
@@ -3746,18 +3746,18 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="176"/>
         <source>Delete</source>
-        <translation>삭제 표시</translation>
+        <translation>삭제</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="364"/>
         <source>Remove</source>
-        <translation>삭제</translation>
+        <translation>제거</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="370"/>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="371"/>
         <source>&lt;font color=&apos;gray&apos;&gt;Qualified removed files will still be automatically aggregated in the next update.&lt;/font&gt;</source>
-        <translation>&lt;font color=&apos;gray&apos;&gt;제거된 파일이 요구에 부합되면 다음번 업데이트 시 여전히 자동 집합됩니다.&lt;/font&gt;</translation>
+        <translation>&lt;font color=&apos;gray&apos;&gt;제거된 파일이 요구에 부합되면 다음 업데이트 시 여전히 자동 집합됩니다.&lt;/font&gt;</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="374"/>
@@ -3772,12 +3772,12 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="378"/>
         <source>Do not ask again</source>
-        <translation>다시 묻지 않기</translation>
+        <translation>다시 알리지 않기</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="445"/>
         <source>Copy</source>
-        <translation>본지에 복사</translation>
+        <translation>복사</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="468"/>
@@ -3828,7 +3828,7 @@ Do you want to delete the link file?</source>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="648"/>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="887"/>
         <source>Sort By</source>
-        <translation>배열 방식</translation>
+        <translation>정렬 방식</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="654"/>
@@ -3846,25 +3846,25 @@ Do you want to delete the link file?</source>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="657"/>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="897"/>
         <source>Newest to oldest</source>
-        <translation>새것부터 오래된 것까지</translation>
+        <translation>최신순</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="657"/>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="897"/>
         <source>Oldest to newest</source>
-        <translation>오래된 것부터 새것까지</translation>
+        <translation>오래된 순</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="660"/>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="900"/>
         <source>Files from large to small</source>
-        <translation>큰것부터 작은 것까지</translation>
+        <translation>크기 내림차순</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="660"/>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="900"/>
         <source>Files from small to large</source>
-        <translation>작은 것부터 큰것까지</translation>
+        <translation>크기 오름차순</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="662"/>
@@ -3894,7 +3894,7 @@ Do you want to delete the link file?</source>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="690"/>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="932"/>
         <source>Sort Preferences</source>
-        <translation>정렬 기본 설정</translation>
+        <translation>정렬 선호</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="693"/>
@@ -3918,7 +3918,7 @@ Do you want to delete the link file?</source>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="728"/>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="977"/>
         <source>Refresh</source>
-        <translation>새 고침</translation>
+        <translation>새로 고침</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="800"/>
@@ -3937,7 +3937,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/controls/menu/idm-actions-manager.cpp" line="990"/>
         <source>Intelligent Space</source>
-        <translation>지능 공간</translation>
+        <translation>지능형 공간</translation>
     </message>
 </context>
 <context>
@@ -3968,17 +3968,17 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/controls/tag-management.cpp" line="123"/>
         <source>Name</source>
-        <translation>블루투스 장치 이름</translation>
+        <translation>표시 이름</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/tag-management.cpp" line="124"/>
         <source>SideBar</source>
-        <translation>사이드바</translation>
+        <translation>가장자리 표시줄</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/tag-management.cpp" line="125"/>
         <source>Menu</source>
-        <translation>메뉴</translation>
+        <translation>오른쪽 마우스 메뉴</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/tag-management.cpp" line="133"/>
@@ -3999,17 +3999,17 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/controls/tag-management.cpp" line="162"/>
         <source>Rename</source>
-        <translation>블루투스 이름 변경</translation>
+        <translation>이름 변경</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/tag-management.cpp" line="169"/>
         <source>Edit Color</source>
-        <translation>색상 편집</translation>
+        <translation>표시 색상 수정</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/tag-management.cpp" line="184"/>
         <source>Delete This Label</source>
-        <translation>이 표식 삭제</translation>
+        <translation>해당 표시 삭제</translation>
     </message>
 </context>
 <context>
@@ -4022,7 +4022,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/controls/navigation-bar/location-bar/location-bar.cpp" line="427"/>
         <source>Search results for all files marked in  &quot;%1&quot;  in &quot;%2&quot;</source>
-        <translation>“%2”에서 모든 “%1” 표식 파일 검색 결과</translation>
+        <translation>“%2”에서 모든 “%1” 표시 파일 검색 결과</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/navigation-bar/location-bar/location-bar.cpp" line="427"/>
@@ -4056,7 +4056,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/mount-operation.cpp" line="214"/>
         <source>Login failed, unknown username or password error, please re-enter!</source>
-        <translation>로그인 실패，알 수 없는 사용자 이름 또는 암호 오류입니다. 다시 입력하십시오！</translation>
+        <translation>로그인 실패, 알 수 없는 사용자 이름 또는 비밀번호 오류입니다. 다시 입력하십시오！</translation>
     </message>
 </context>
 <context>
@@ -4064,12 +4064,12 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/controls/navigation-bar/navigation-tool-bar.cpp" line="35"/>
         <source>Go Back</source>
-        <translation>돌아가다</translation>
+        <translation>뒤로 가기</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/navigation-bar/navigation-tool-bar.cpp" line="39"/>
         <source>Go Forward</source>
-        <translation>앞으로 가다</translation>
+        <translation>앞으로 가기</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/navigation-bar/navigation-tool-bar.cpp" line="43"/>
@@ -4084,12 +4084,12 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/controls/navigation-bar/navigation-tool-bar.cpp" line="88"/>
         <source>Cd Up</source>
-        <translation>위로 가다</translation>
+        <translation>위로</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/navigation-bar/navigation-tool-bar.cpp" line="94"/>
         <source>Refresh</source>
-        <translation>새 고침</translation>
+        <translation>새로 고침</translation>
     </message>
 </context>
 <context>
@@ -4112,12 +4112,12 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/controls/property-page/open-with-properties-page.cpp" line="260"/>
         <source>OK</source>
-        <translation>나중에 재시작</translation>
+        <translation>확인</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/property-page/open-with-properties-page.cpp" line="264"/>
         <source>Cancel</source>
-        <translation>실행 취소</translation>
+        <translation>취소</translation>
     </message>
 </context>
 <context>
@@ -4215,7 +4215,7 @@ Do you want to delete the link file?</source>
         <location filename="../../libpeony-qt/controls/property-page/permissions-properties-page.cpp" line="643"/>
         <location filename="../../libpeony-qt/controls/property-page/permissions-properties-page.cpp" line="669"/>
         <source>Permission refinement settings</source>
-        <translation>권한 세부 설정</translation>
+        <translation>세분 권한 설정</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/property-page/permissions-properties-page.cpp" line="670"/>
@@ -4250,7 +4250,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/controls/property-page/basic-properties-page.cpp" line="1813"/>
         <source>Whether to apply to the current selected option or to selected options and subfolders and subfiles.</source>
-        <translation>현재 선택한 항목에만 적용하시겠습니까, 아니면 선택한 항목과 하위 폴더 및 파일에도 적용하시겠습니까.</translation>
+        <translation>현재 선택한 항목에만 적용하시겠습니까, 아니면 선택한 항목과 하위 폴더 및 하위 파일에 모두 적용하시겠습니까</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/property-page/basic-properties-page.cpp" line="1824"/>
@@ -4265,12 +4265,12 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/controls/property-page/basic-properties-page.cpp" line="1848"/>
         <source>Ok</source>
-        <translation>괜찮습니다</translation>
+        <translation>확인</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/property-page/basic-properties-page.cpp" line="1849"/>
         <source>Cancel</source>
-        <translation>실행 취소</translation>
+        <translation>취소</translation>
     </message>
 </context>
 <context>
@@ -4278,7 +4278,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/windows/properties-window.cpp" line="382"/>
         <source>Trash</source>
-        <translation>삭제</translation>
+        <translation>휴지통</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/windows/properties-window.cpp" line="386"/>
@@ -4318,17 +4318,17 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/windows/properties-window.cpp" line="602"/>
         <source>Ok</source>
-        <translation>괜찮습니다</translation>
+        <translation>확인</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/windows/properties-window.cpp" line="603"/>
         <source>Cancel</source>
-        <translation>실행 취소</translation>
+        <translation>취소</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/windows/properties-window.cpp" line="613"/>
         <source>Restore</source>
-        <translation>열기</translation>
+        <translation>복원</translation>
     </message>
 </context>
 <context>
@@ -4336,7 +4336,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/controls/property-page/recent-and-trash-properties-page.cpp" line="144"/>
         <source>Show confirm dialog while trashing.</source>
-        <translation>휴지통으로 삭제할 때 확인 창을 띄웁니다.</translation>
+        <translation>휴지통으로 삭제 시 확인 대화 상자를 팝업합니다.</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/property-page/recent-and-trash-properties-page.cpp" line="175"/>
@@ -4359,12 +4359,12 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/controls/property-page/recent-and-trash-properties-page.cpp" line="109"/>
         <source>Name</source>
-        <translation>블루투스 장치 이름</translation>
+        <translation>이름</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/property-page/recent-and-trash-properties-page.cpp" line="110"/>
         <source>Location</source>
-        <translation>위치 정보 허용</translation>
+        <translation>위치</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/property-page/recent-and-trash-properties-page.cpp" line="259"/>
@@ -4412,12 +4412,12 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/controls/tool-bar/search-bar-container.cpp" line="248"/>
         <source>Clear</source>
-        <translation>클립보드 비우기</translation>
+        <translation>비우기</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/tool-bar/search-bar-container.h" line="166"/>
         <source>all</source>
-        <translation>전체 선택</translation>
+        <translation>전체</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/tool-bar/search-bar-container.h" line="166"/>
@@ -4492,7 +4492,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/model/side-bar-favorite-item.cpp" line="84"/>
         <source>Trash</source>
-        <translation>삭제</translation>
+        <translation>휴지통</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/model/side-bar-favorite-item.cpp" line="87"/>
@@ -4502,7 +4502,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/model/side-bar-favorite-item.cpp" line="95"/>
         <source>Quick Access</source>
-        <translation>빠른 액세스</translation>
+        <translation>빠른 접근</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/model/side-bar-favorite-item.cpp" line="198"/>
@@ -4536,7 +4536,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/controls/menu/side-bar-menu/side-bar-menu.cpp" line="161"/>
         <source>Remove from bookmark</source>
-        <translation>“빠른 방문”에서 고정 해제</translation>
+        <translation>“빠른 접근”에서 고정 해제</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/menu/side-bar-menu/side-bar-menu.cpp" line="274"/>
@@ -4571,12 +4571,12 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/model/side-bar-model.cpp" line="98"/>
         <source>Network</source>
-        <translation>네트워크 련결</translation>
+        <translation>네트워크</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/model/side-bar-model.cpp" line="255"/>
         <source>Create</source>
-        <translation>새로운 보호상자</translation>
+        <translation>새로 만들기</translation>
     </message>
 </context>
 <context>
@@ -4630,7 +4630,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/sync-thread.cpp" line="66"/>
         <source>Notify</source>
-        <translation>참고 안내</translation>
+        <translation>안내</translation>
     </message>
 </context>
 <context>
@@ -4638,7 +4638,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/controls/tag-management.cpp" line="397"/>
         <source>label management</source>
-        <translation>표식 관리</translation>
+        <translation>표시 관리</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/tag-management.cpp" line="413"/>
@@ -4651,7 +4651,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/controls/tool-bar/tool-bar.cpp" line="139"/>
         <source>Sort Type</source>
-        <translation>배렬 유형</translation>
+        <translation>정렬 유형</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/tool-bar/tool-bar.cpp" line="141"/>
@@ -4671,7 +4671,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/controls/tool-bar/tool-bar.cpp" line="144"/>
         <source>Modified Date</source>
-        <translation>수정한 날짜</translation>
+        <translation>수정 날짜</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/tool-bar/tool-bar.cpp" line="73"/>
@@ -4686,18 +4686,18 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/controls/tool-bar/tool-bar.cpp" line="161"/>
         <source>Ascending</source>
-        <translation>오름 차순</translation>
+        <translation>오름차순</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/tool-bar/tool-bar.cpp" line="157"/>
         <source>Descending</source>
-        <translation>내림 차순</translation>
+        <translation>내림차순</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/tool-bar/tool-bar.cpp" line="191"/>
         <location filename="../../libpeony-qt/controls/tool-bar/tool-bar.cpp" line="336"/>
         <source>Copy</source>
-        <translation>본지에 복사</translation>
+        <translation>복사</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/tool-bar/tool-bar.cpp" line="194"/>
@@ -4722,12 +4722,12 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/controls/tool-bar/tool-bar.cpp" line="222"/>
         <source>Restore</source>
-        <translation>열기</translation>
+        <translation>복원</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/tool-bar/tool-bar.cpp" line="271"/>
         <source>Options</source>
-        <translation>선택항</translation>
+        <translation>옵션</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/tool-bar/tool-bar.cpp" line="274"/>
@@ -4747,7 +4747,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/controls/tool-bar/tool-bar.cpp" line="296"/>
         <source>Let the program still run after closing the last window. This will reduce the time for the next launch, but it will also consume resources in backend.</source>
-        <translation>파일 관리자를 모든 창을 닫은 후에도 계속 실행합니다. 이는 다음 시작 시간을 단축하지만, 지속적으로 리소스를 점유하게 됩니다.</translation>
+        <translation>파일 관리자를 모든 창을 닫은 후에도 계속 실행합니다. 이는 다음 시작 시간을 단축하지만, 지속적으로 자원을 점유하게 됩니다.</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/tool-bar/tool-bar.cpp" line="308"/>
@@ -4762,7 +4762,7 @@ Do you want to delete the link file?</source>
         <location filename="../../libpeony-qt/tooltips-manager.cpp" line="318"/>
         <location filename="../../libpeony-qt/tooltips-manager.cpp" line="490"/>
         <source>Unknow</source>
-        <translation>알 수 없음</translation>
+        <translation>미지</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/tooltips-manager.cpp" line="235"/>
@@ -4775,7 +4775,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/tooltips-manager.cpp" line="239"/>
         <source>Duration：</source>
-        <translation>재생 시간:</translation>
+        <translation>지속 시간:</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/tooltips-manager.cpp" line="241"/>
@@ -4783,7 +4783,7 @@ Do you want to delete the link file?</source>
         <location filename="../../libpeony-qt/tooltips-manager.cpp" line="322"/>
         <location filename="../../libpeony-qt/tooltips-manager.cpp" line="493"/>
         <source>Size: </source>
-        <translation>파일 크기: </translation>
+        <translation>크기: </translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/tooltips-manager.cpp" line="250"/>
@@ -4818,7 +4818,7 @@ Do you want to delete the link file?</source>
         <location filename="../../libpeony-qt/usershare-manager.cpp" line="115"/>
         <location filename="../../libpeony-qt/usershare-manager.cpp" line="148"/>
         <source>Warning</source>
-        <translation>알림</translation>
+        <translation>경고</translation>
     </message>
 </context>
 <context>
@@ -4875,20 +4875,20 @@ Do you want to delete the link file?</source>
         <location filename="../../libpeony-qt/controls/directory-view/directory-view-factory/icon-view-factory.h" line="46"/>
         <location filename="../../libpeony-qt/controls/directory-view/directory-view-factory/icon-view-factory.h" line="99"/>
         <source>Show the folder children as icons.</source>
-        <translation>아이콘 형태로 디렉토리 보기.</translation>
+        <translation>아이콘 형태로 폴더 보기.</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/directory-view/directory-view-factory/list-view-factory.h" line="42"/>
         <location filename="../../libpeony-qt/controls/directory-view/directory-view-factory/list-view-factory.h" line="62"/>
         <location filename="../../libpeony-qt/controls/directory-view/directory-view-factory/list-view-factory.h" line="95"/>
         <source>List View</source>
-        <translation>라스트 보기</translation>
+        <translation>리스트 보기</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/directory-view/directory-view-factory/list-view-factory.h" line="48"/>
         <location filename="../../libpeony-qt/controls/directory-view/directory-view-factory/list-view-factory.h" line="101"/>
         <source>Show the folder children as rows in a list.</source>
-        <translation>디렉토리를 리스트 형식으로 표시합니다.</translation>
+        <translation>폴더를 리스트 형식으로 표시합니다.</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/controls/property-page/basic-properties-page-factory.h" line="40"/>
@@ -4926,7 +4926,7 @@ Do you want to delete the link file?</source>
         <location filename="../../libpeony-qt/convenient-utils/file-operation-utils.cpp" line="226"/>
         <location filename="../../libpeony-qt/convenient-utils/file-operation-utils.cpp" line="509"/>
         <source>Delete</source>
-        <translation>삭제 표시</translation>
+        <translation>삭제</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/convenient-utils/file-operation-utils.cpp" line="398"/>
@@ -4936,12 +4936,12 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/convenient-utils/file-operation-utils.cpp" line="399"/>
         <source>Files that are emptied cannot be recovered</source>
-        <translation>비운 파일은 회복할수 없습니다</translation>
+        <translation>비운 파일은 복원할 수 없습니다</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/convenient-utils/file-operation-utils.cpp" line="400"/>
         <source>Clean</source>
-        <translation>모두 삭제</translation>
+        <translation>비우기</translation>
     </message>
     <message>
         <source>Clean the Trash</source>
@@ -5004,7 +5004,7 @@ Do you want to delete the link file?</source>
     <message>
         <location filename="../../libpeony-qt/windows/format_dialog.cpp" line="1142"/>
         <source>Close</source>
-        <translation>끄기</translation>
+        <translation>닫기</translation>
     </message>
     <message>
         <location filename="../../libpeony-qt/windows/format_dialog.cpp" line="1103"/>
diff --git a/translations/peony-qt-desktop/peony-qt-desktop_ko_KP.ts b/translations/peony-qt-desktop/peony-qt-desktop_ko_KP.ts
index 547d413..1009831 100644
--- a/translations/peony-qt-desktop/peony-qt-desktop_ko_KP.ts
+++ b/translations/peony-qt-desktop/peony-qt-desktop_ko_KP.ts
@@ -25,12 +25,12 @@
     <message>
         <location filename="../../peony-qt-desktop/controls/file-action-controler.cpp" line="511"/>
         <source>管理插件</source>
-        <translation></translation>
+        <translation>플러그인 관리</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/controls/file-action-controler.cpp" line="512"/>
         <source>调整插件位置</source>
-        <translation></translation>
+        <translation>플러그인 위치 조정</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/controls/file-action-controler.cpp" line="556"/>
@@ -334,7 +334,7 @@
     <message>
         <location filename="../../peony-qt-desktop/peony-desktop-application.cpp" line="542"/>
         <source>Add desktop widget to desktop</source>
-        <translation></translation>
+        <translation>Add desktop widget to desktop</translation>
     </message>
 </context>
 <context>
@@ -351,7 +351,7 @@
     <message>
         <location filename="../../peony-qt-desktop/qml-desktop/platformwindow.cpp" line="3536"/>
         <source>Folder</source>
-        <translation>문서함</translation>
+        <translation>폴더</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/qml-desktop/platformwindow.cpp" line="3539"/>
@@ -530,7 +530,7 @@ Install widget packages to get started.</source>
     <message>
         <location filename="../../peony-qt-desktop/desktop-widgets/base/desktop-widgets-menu.h" line="69"/>
         <source>Widget Menu</source>
-        <translation></translation>
+        <translation>Widget Menu</translation>
     </message>
 </context>
 </TS>
